### ISSUE TITLE
Update WDL headers

### ISSUE THREAD

**Issue by mwalker174** [broadinstitute/gatk-sv#7]
These headers can be deleted from all WDLs:
```
##########################################################################################

## Base script:   https://portal.firecloud.org/#methods/Talkowski-SV/00_batch_evidence_merging/15/wdl

## Github commit: talkowski-lab/gatk-sv-v1:<ENTER HASH HERE IN FIRECLOUD>

##########################################################################################
```
For consistency, we should add new headers to all top-level WDLs (`GATKSVPipeline*.wdl`, `Module0*.wdl`, etc) and adopt the GATK style, e.g. [joint genotyping wdl](https://portal.firecloud.org/?return=terra#methods/gatk/joint-discovery-gatk4/14/wdl). Use the description information from the README for each workflow.

**Comment by epiercehoffman** [comment]
Partially addressed by removing outdated headers in #160

**Comment by mwalker174** [comment]
All headers have been removed, so closing this issue in favor of providing relevant info in the documentation.
